src/components/selectorComponents/TextInput.tsx

```diff
@@ -1,18 +1,43 @@
-import React from "react";
+import React, { useState } from "react";
 import styles from "./TextInput.module.css";
 import classNames from "classnames";
 
 interface TextInputProps {
-  placeholder?: string;
+  placeholder: string;
 }
 
 const TextInput = (props: TextInputProps) => {
+  const [currentValue, setCurrentValue] = useState("");
+  const [error, setError] = useState(false)
```

Comment:
**nitpick:** It's better to set error as string, so you can control it instead of hardcoding it